cli/asc.js

```diff
@@ -380,6 +380,8 @@ exports.main = function main(argv, options, callback) {
     const filename = argv[i];
 
     let sourcePath = String(filename).replace(/\\/g, "/").replace(/(\.ts|\/)$/, "");
+    // Setting the path to relative path
+    sourcePath = sourcePath.indexOf(baseDir) == 0 ? sourcePath.slice(baseDir.length) : sourcePath;
```

Comment:
This somehow feels like half a fix for paths inside `baseDir` only. Or does it work for other absolute paths outside `baseDir` properly already?

Reply:
I missed this test case.  Now I fixed it. 

What do you think the way to change implementation of [`readFileNode`](https://github.com/LiaoPeng/assemblyscript/blob/ab94853a23dbf33451f64c6ca631cc6cfad6eb3d/cli/asc.js#L727) like this?

```
function readFileNode(filename, baseDir) {
    try {
      let text;
      stats.readCount++;
      stats.readTime += measure(() => {
        let filePath = path.isAbsolute(filename) ? filename :  path.join(baseDir, filename);
        text = fs.readFileSync(filePath , { encoding: "utf8" });
      });
      return text;
    } catch (e) {
      return null;
    }
  }
```